Resources/Prototypes/Recipes/Construction/Graphs/weapons/shotgunsawn.yml

```diff
@@ -0,0 +1,12 @@
+- type: constructionGraph
+  id: ShotgunSawn
+  start: shotgundoublebarreled
+  graph:
+    - node: shotgundoublebarreled
+      edges:
+        - to: shotgunsawn
+          steps:
+            - tool: Sawing
+              doAfter: 2
+    - node: shotgunsawn
+      entity: WeaponShotgunSawn
```

Comment:
so one issue here is that if the gun is currently loaded, it'll delete all the bullets in it when converting into a sawn-off. so you probably want some igraphcondition making sure the gun is empty first. if you're up to it in this PR you can add that, otherwise just tell me and I can merge this and do it in a later PR

Reply:
plz merge